### ISSUE TITLE
[BUG] Failure to build

### ISSUE THREAD

**Issue by planetA** [jetli/create-yew-app#9]
Hello,

when I try to start the sample application, the build process fails:

```
 $ npm init yew-app ask.yew
 $ npm start

> yew-app@0.1.0 start <path>/ask.yew
> WASM_PACK_PROFILE=dev parcel static/index.html -p 8000 --open

Server running at http://localhost:8000 
🚨  <path>/ask.yew/node_modules/parcel-plugin-wasm.rs/wasm-loader.js:2:14: Cannot resolve dependency './yew_app_bg.js' at '<path>/ask.yew/node_modules/parcel-plugin-wasm.rs/yew_app_bg.js'
  1 | var wasm;const __exports = {};
> 2 | export * from "./yew_app_bg.js";
    |              ^
  3 | 
  4 |       function init(wasm_path) {
  5 |           const fetchPromise = fetch(wasm_path);

```

My application is called "ask.yew"

**Comment by jetli** [comment]
temp fix by `wasm-bindgen = "0.2, <0.2.61"`

**Comment by patientplatypus6** [comment]
This results in the following error - 

```
sudo npm start

> yew-app@0.1.0 start
> cross-env WASM_PACK_PROFILE=dev parcel static/index.html -p 8000 --open

Server running at http://localhost:8000
🚨  /Users/peterweyand/Code/rustprojects/gitapp/Cargo.toml: Command failed: wasm-pack build --dev
Error: Error during execution of `cargo metadata`:     Updating crates.io index
error: failed to select a version for `wasm-bindgen`.
    ... required by package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`
versions that meet the requirements `^0.2, <0.2.61` are: 0.2.60, 0.2.59, 0.2.58, 0.2.57, 0.2.56, 0.2.55, 0.2.54, 0.2.53, 0.2.52, 0.2.51, 0.2.50, 0.2.49, 0.2.48, 0.2.47, 0.2.46, 0.2.45, 0.2.44, 0.2.43, 0.2.42, 0.2.41, 0.2.40, 0.2.39, 0.2.38, 0.2.37, 0.2.36, 0.2.35, 0.2.34, 0.2.33, 0.2.32, 0.2.31, 0.2.30, 0.2.29, 0.2.28, 0.2.27, 0.2.26, 0.2.25, 0.2.24, 0.2.23, 0.2.22, 0.2.21, 0.2.20, 0.2.19, 0.2.18, 0.2.17, 0.2.16, 0.2.15, 0.2.14, 0.2.13, 0.2.12, 0.2.11, 0.2.10, 0.2.9, 0.2.8, 0.2.7, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0

the package `wasm-bindgen` links to the native library `wasm_bindgen`, but it conflicts with a previous package which links to `wasm_bindgen` as well:
package `wasm-bindgen-shared v0.2.82`
    ... which satisfies dependency `wasm-bindgen-shared = "=0.2.82"` of package `wasm-bindgen-backend v0.2.82`
    ... which satisfies dependency `wasm-bindgen-backend = "=0.2.82"` of package `wasm-bindgen-macro-support v0.2.82`
    ... which satisfies dependency `wasm-bindgen-macro-support = "=0.2.82"` of package `wasm-bindgen-macro v0.2.82`
    ... which satisfies dependency `wasm-bindgen-macro = "=0.2.82"` of package `wasm-bindgen v0.2.82`
    ... which satisfies dependency `wasm-bindgen = "^0.2.37"` of package `console_error_panic_hook v0.1.7`
    ... which satisfies dependency `console_error_panic_hook = "^0.1"` of package `wasm-bindgen-test v0.3.31`
    ... which satisfies dependency `wasm-bindgen-test = "^0.3"` of package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the links ='wasm-bindgen' value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

all possible versions conflict with previously selected packages.

  previously selected package `wasm-bindgen v0.2.82`
    ... which satisfies dependency `wasm-bindgen = "^0.2.37"` of package `console_error_panic_hook v0.1.7`
    ... which satisfies dependency `console_error_panic_hook = "^0.1"` of package `wasm-bindgen-test v0.3.31`
    ... which satisfies dependency `wasm-bindgen-test = "^0.3"` of package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`

failed to select a version for `wasm-bindgen` which could resolve this conflict


Error: Error during execution of `cargo metadata`:     Updating crates.io index
error: failed to select a version for `wasm-bindgen`.
    ... required by package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`
versions that meet the requirements `^0.2, <0.2.61` are: 0.2.60, 0.2.59, 0.2.58, 0.2.57, 0.2.56, 0.2.55, 0.2.54, 0.2.53, 0.2.52, 0.2.51, 0.2.50, 0.2.49, 0.2.48, 0.2.47, 0.2.46, 0.2.45, 0.2.44, 0.2.43, 0.2.42, 0.2.41, 0.2.40, 0.2.39, 0.2.38, 0.2.37, 0.2.36, 0.2.35, 0.2.34, 0.2.33, 0.2.32, 0.2.31, 0.2.30, 0.2.29, 0.2.28, 0.2.27, 0.2.26, 0.2.25, 0.2.24, 0.2.23, 0.2.22, 0.2.21, 0.2.20, 0.2.19, 0.2.18, 0.2.17, 0.2.16, 0.2.15, 0.2.14, 0.2.13, 0.2.12, 0.2.11, 0.2.10, 0.2.9, 0.2.8, 0.2.7, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0

the package `wasm-bindgen` links to the native library `wasm_bindgen`, but it conflicts with a previous package which links to `wasm_bindgen` as well:
package `wasm-bindgen-shared v0.2.82`
    ... which satisfies dependency `wasm-bindgen-shared = "=0.2.82"` of package `wasm-bindgen-backend v0.2.82`
    ... which satisfies dependency `wasm-bindgen-backend = "=0.2.82"` of package `wasm-bindgen-macro-support v0.2.82`
    ... which satisfies dependency `wasm-bindgen-macro-support = "=0.2.82"` of package `wasm-bindgen-macro v0.2.82`
    ... which satisfies dependency `wasm-bindgen-macro = "=0.2.82"` of package `wasm-bindgen v0.2.82`
    ... which satisfies dependency `wasm-bindgen = "^0.2.37"` of package `console_error_panic_hook v0.1.7`
    ... which satisfies dependency `console_error_panic_hook = "^0.1"` of package `wasm-bindgen-test v0.3.31`
    ... which satisfies dependency `wasm-bindgen-test = "^0.3"` of package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the links ='wasm-bindgen' value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

all possible versions conflict with previously selected packages.

  previously selected package `wasm-bindgen v0.2.82`
    ... which satisfies dependency `wasm-bindgen = "^0.2.37"` of package `console_error_panic_hook v0.1.7`
    ... which satisfies dependency `console_error_panic_hook = "^0.1"` of package `wasm-bindgen-test v0.3.31`
    ... which satisfies dependency `wasm-bindgen-test = "^0.3"` of package `yew-app v0.1.0 (/Users/peterweyand/Code/rustprojects/gitapp)`

failed to select a version for `wasm-bindgen` which could resolve this conflict


    at ChildProcess.exithandler (node:child_process:399:12)
    at ChildProcess.emit (node:events:520:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
^C%                                  
```
